### DISCUSSION TITLE
Custom CSS for specific form field

### DISCUSSION THREAD

**Discussion by dkmbuild** [Budibase/budibase#13194] [Help] [answered]
Looking for a way to apply bold to a specific form field name.

![bold specific form field](https://github.com/Budibase/budibase/assets/149087656/8bf11194-7c38-45be-9945-30c16238cf85)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @dkmbuild,

You likely won't be able to target a specific field. Those svelte-zgf5va get automatically generated on page load so they will change on every refresh.

You can target all labels by using the [embed](https://docs.budibase.com/docs/embed) component and putting it at the top of your component tree. From here then pasting in your style code, minus the auto generated class at the end. That should work.

From the looks of your screenshot you are trying to apply these styles to the form component rather than the component itself. This will only apply to the form and is just apply as inline styling. So in this case your style+classes won't even work in this instance. As it would look like the below which isn't valid inline styling.

`<form style="<style>.spectrium-FieldLabel.....">`

Hopefully this makes sense and is helpful.
